simulators/nav/src/store/modules/simulatorState.ts

```diff
@@ -115,6 +118,8 @@ const getters = {
 
   roverPath: (simState:SimulatorState):Odom[] => simState.path,
 
+  FOVAreaPath: (simState:SimulatorState):Path2D => simState.FOVAreaPath,
```

Comment:
is this something that can be set on the simulator gui?

Reply:
This contains the area on the map that the rover's FOV has seen. Do you mean if we can set this area pre-simulation?

Reply:
yeah that's what i mean, can how wide the FOV and depth of it be changed?

Reply:
Yep

https://user-images.githubusercontent.com/71047773/157580287-e600bd19-d701-49ae-aaed-93a1e92724f4.mov